ingredient/ingredient-kubernetes/src/functions.ts

```diff
@@ -0,0 +1,12 @@
+import {BaseUtility, IngredientManager} from '@azbake/core'
+
+export class MyUtils extends BaseUtility {
+
+    public my_function() {

```

Comment:
This is boilerplate code. Do you have any ideas on your functions? Namespace concatentation, value or state reflection, etc?

Reply:
Thanks! There will be token replacement, but I haven't looked into that yet. Not sure if there's stuff in the core helpers that will do that without my ingredient needing any functions.ts (in which case this probably just gets deleted), or if I'll have to have something here.

Reply:
functions.ts is for exposing functions to your yaml recipe definition. For the token replacement we'll want to handle that internal of the ingredient (can scope it into it's own class/source). Don't think we have any util functions for k8s so probably can remove it for now

Reply:
Thanks, deleted file. _For now..._